crates/db-arch/Cargo.toml

```diff
@@ -14,6 +14,7 @@ kvm-bindings = { version = ">=0.5.0", features = ["fam-wrappers"] }
 kvm-ioctls = ">=0.9.0"
 vm-memory = { version = "0.7", features = ["backend-mmap"] }
 vmm-sys-util = ">=0.8.0"
+db-boot = { path = "../db-boot" }
```

Comment:
After db-boot is published in crate.io, this part will be changed.

Reply:
"db-boot = { path = "../db-boot", version = "0.1.0" } "
will work for both cases